scss/mixins/_type-mixins.scss

```diff
@@ -80,3 +80,7 @@
   font-weight: bold;
   padding: 0 0.3em;
 }
+
+@mixin t-upper {
```

Comment:
I was thinking I would add documentation for it (but there are none for the `t-` mixins) and was the under the impression that was the norm. Was also unsure about the ettiquette using `@extend` for classes; some folks don't like `@extend`.

Anyway, I'll remove it, and while I'm at it, how about I do the same with `t-highlight`, and replace the `@include` usages with `@extend`?